pyscriptjs/src/plugins/stdiodirector.ts

```diff
@@ -0,0 +1,28 @@
+import { Plugin } from "../plugin";
+import { TargetedStdio, StdioMultiplexer } from "../stdio";
+
+export class StdioDirector extends Plugin {
```

Comment:
Maybe adding docstrings/comments to this class might be a good idea since we are adding an event listener and then removing it once `beforePyScriptExec` is called. What do you think? 🤔